UnityProject/Assets/Scripts/Shuttles/ShuttleConsole.cs

```diff
@@ -72,7 +72,11 @@ private IEnumerator InitMatrixMove()
 					Category.Matrix);
 			}
 
-			if (ShuttleMatrixMove != null)
+			if (ShuttleMatrixMove.NoConsole == true)
```

Comment:
```suggestion
			if (ShuttleMatrixMove.NoConsole)
```
When checking a bool, you don't really need to compare it to true.